app/views/layouts/application.html.erb

```diff
@@ -99,6 +99,7 @@
 
 <% content_for :body_end do %>
   <%= content_tag :div, nil, class: "js-AsyncGA", data: ga_tracking_data %>
+  <%= content_tag :div, nil, class: "js-Sentry", data: {sentry_js_dsn: config_item(:sentry_js_dsn)} %>
```

Comment:
Minor styling issue, either remove the `{}` or add a space after / before the braces